internal/pkg/rerere/rerere.go

```diff
@@ -177,19 +200,25 @@ func checkContexts(log *logrus.Entry, ghc githubClient, contexts prowflagutil.St
 	// List all check runs.
 	checkRun, err := ghc.ListCheckRuns(org, repo, lastCommitRef)
 	if err != nil {
-		return fmt.Errorf("list %s check runs failed: %v", retestingBranch, err)
+		return false, fmt.Errorf("list %s check runs failed: %v", retestingBranch, err)
 	}
 	for _, runs := range checkRun.CheckRuns {
 		if runs.Status == checkRunStatusCompleted {
-			log.Infof("%s runs passed.", runs.Name)
-			passedContexts.Insert(runs.Name)
+			if runs.Conclusion == checkRunConclusionNeutral || runs.Conclusion == checkRunConclusionSuccess {
+				log.Infof("%s runs passed.", runs.Name)
+				passedContexts.Insert(runs.Name)
+			} else if contextsSet.Has(runs.Name) {
```

Comment:
In addition to neutral and success, what other conclusions are there? If the types of conclusions are relatively certain and the number is relatively small, I think it will be easier to understand to list them in the conditions.

Reply:
Can be one of success, failure, neutral, cancelled, skipped, timed_out, or action_required.

Reply:
Understand, this seems too much to list.